src/diagnostics/LogPointsManager.ts

```diff
@@ -198,6 +205,23 @@ export class LogPointsManager extends vscode.Disposable {
         debugSessionManager.recoverLogpoints(documentUri);
     }
 
+    private onDebugSessionClose(debugSession: vscode.DebugSession): void {
+
+        const debugSessionManager = this._debugSessionManagerMapping[debugSession.id];
+
+        if (!debugSessionManager) {
+            // If debugSessionManager does not exist, it might have been handled already.
+            return;
+        }
+
+        if (!vscode.window.activeTextEditor) {
+            return;
+        }
+        const documentUri = vscode.window.activeTextEditor.document.uri;
```

Comment:
Is there a scenario where the document is undefined or uri is undefined?

Reply:
No. An active text editor has to have a document uri.